### PR TITLE
tempita: fix broken repo

### LINKED ISSUE #79207: tempita
**Describe the bug**
[Source git repo](https://github.com/gjhiggins/tempita) for the current derivation was removed living the derivation broken. Now you will get a 404 error each time you try to build this derivation.

**To Reproduce**
1. Build tempita derivation.

**Expected behavior**
Tempita should be built.

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

 - system: `"x86_64-darwin"`
 - host os: `Darwin 18.7.0, macOS 10.14.6`
 - multi-user?: `no`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.3.1`
 - channels(juan_pablo_gomez): `"nixpkgs-20.03pre207234.ec29bb50bf4"`
 - nixpkgs: `/Users/juan_pablo_gomez/.nix-defexpr/channels/nixpkgs`

### PR DESCRIPTION
**Pull request by jpgu-epam**
###### Motivation for this change
Tempita src is broken.

Closes #79207 

###### Things done
Update tempita src to a new existing repo.

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/tempita/default.nix [hunk 1]
```diff
@@ -1,20 +1,18 @@
-{ lib, buildPythonPackage, fetchFromGitHub, nose }:
+{ lib, buildPythonPackage, fetchurl, nose }:
 
 buildPythonPackage {
   version = "0.5.3-2016-09-28";
```
Comment (DzmitrySudnik) [inline pkgs/development/python-modules/tempita/default.nix]: It's "0.5.3" in the sources: https://bitbucket.org/ianb/tempita/src/97392d008cc8/setup.py

**Review by DzmitrySudnik — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/tempita/default.nix [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
-  version = "0.5.3-2016-09-28";
+  version = "0.5.3";
   pname = "tempita";
 
-  src = fetchFromGitHub {
-    owner = "gjhiggins";
-    repo = "tempita";
-    rev = "47414a7c6e46a9a9afe78f0bce2ea299fa84d10";
-    sha256 = "0f33jjjs5rvp7ar2j6ggyfykcrsrn04jaqcq71qfvycf6b7nw3rn";
+  src = fetchurl {
```
Comment (veprbl) [inline pkgs/development/python-modules/tempita/default.nix]: Why not `fetchFromBitbucket`?

File: pkgs/development/python-modules/tempita/default.nix [hunk 3]
```diff
@@ -1,20 +1,18 @@
...
+    url = https://bitbucket.org/ianb/tempita/get/97392d008cc8.tar.gz;
+    sha256 = "0nxnkxjvfyxygmws2zxql590mwqsqd1rnhy80m9nbpdh81p7vh9y";
   };
 
   buildInputs = [ nose ];
 
   meta = {
-    homepage = https://github.com/gjhiggins/tempita;
+    homepage = http://pythonpaste.org/tempita/;
```
Comment (jonringer) [inline pkgs/development/python-modules/tempita/default.nix]:
```suggestion
    homepage = "http://pythonpaste.org/tempita/";
```

File: pkgs/development/python-modules/tempita/default.nix (hunk 2 above)
Comment (jonringer) [reply pkgs/development/python-modules/tempita/default.nix]:
```
fetchFromBitbucket {
  owner = "ianb";
  repo = pname;
  rev = "97392d008cc8";
  sha256 = "0nxnkxjvfyxygmws2zxql590mwqsqd1rnhy80m9nbpdh81p7vh9a";
};
```

Comment (jpgu-epam) [reply pkgs/development/python-modules/tempita/default.nix]: Will do it tomorrow, thanks.

File: pkgs/development/python-modules/tempita/default.nix [hunk 4]
```diff
@@ -1,20 +1,18 @@
-{ lib, buildPythonPackage, fetchFromGitHub, nose }:
+{ lib, buildPythonPackage, fetchurl, nose }:
 
 buildPythonPackage {
-  version = "0.5.3-2016-09-28";
+  version = "0.5.3";
   pname = "tempita";
```
Comment (jonringer) [inline pkgs/development/python-modules/tempita/default.nix]:
```suggestion
  pname = "tempita";
  disabled = isPy3k;
```

```
builder for '/nix/store/dpkdcv6188hc0x19492zqcmarvy2sd08-python3.7-tempita-0.5.3.drv' failed with exit code 1; last 10 log lines:
      except SyntaxError, e:
                        ^
  SyntaxError: invalid syntax

  ----------------------------------------------------------------------
  Ran 1 test in 0.005s

  FAILED (errors=1)
  Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
  error: Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
cannot build derivation '/nix/store/4aisfif26rqqmcmab0q1dh5xi4hp81f5-python3.7-sqlalchemy-migrate-0.12.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/wjx2cmyj4svdpvnsxjvcl15j9wqgz8zz-python3.7-buildbot-2.6.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/kmnzp31ppds2n7fxj6k4y89yww48f67f-python3.7-python3.7-buildbot-2.6.0-with-plugins.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/sy3b14k6s2a4ng7jkjyy7gagfi0km04w-python3.7-python3.7-buildbot-2.6.0-with-plugins.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/983azjq42fa334wrzb59zbqalcrhs6hx-pytrainer-2.0.1.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/x9bhhd2cl252zfgaxzj3syjig3xm4xki-env.drv': 11 dependencies couldn't be built
[5 built (2 failed), 49 copied (184.7 MiB), 35.4 MiB DL]
error: build of '/nix/store/x9bhhd2cl252zfgaxzj3syjig3xm4xki-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/79209
11 package failed to build:
buildbot buildbot-full buildbot-ui python37Packages.sqlalchemy_migrate python37Packages.tempita python38Packages.buildbot python38Packages.buildbot-full python38Packages.buildbot-ui python38Packages.sqlalchemy_migrate python38Packages.tempita pytrainer

2 package built:
python27Packages.sqlalchemy_migrate python27Packages.tempita
```

File: pkgs/development/python-modules/tempita/default.nix [hunk 5]
```diff
@@ -1,20 +1,18 @@
-{ lib, buildPythonPackage, fetchFromGitHub, nose }:
+{ lib, buildPythonPackage, fetchurl, nose }:
```
Comment (jonringer) [inline pkgs/development/python-modules/tempita/default.nix]:
```suggestion
{ lib, buildPythonPackage, isPy3k, fetchurl, nose }:
```

**Comment by aanderse** [comment]
ping @jpgu-epam

File: pkgs/development/python-modules/tempita/default.nix (hunk 2 above)
Comment (doronbehar) [reply pkgs/development/python-modules/tempita/default.nix:7]: > Will do it tomorrow, thanks.

Tomorrow was long ago. @jpgu-epam tempita is still broken on master, is there any chance you could reach to it?

**Pull request closed without merging**